packages/lisk-p2p/src/p2p.ts

```diff
@@ -15,14 +15,20 @@
 
 import { EventEmitter } from 'events';
 import * as http from 'http';
+// tslint:disable-next-line no-require-imports
+import shuffle = require('lodash.shuffle');
```

Comment:
why don't we do something like: `import { shuffle } from 'lodash';`?

Reply:
It was used in `PeerPool` like this and I just moved it here, but you are right I will change it to `from` syntax and update `package.json`

Reply:
@SargeKhan @ishantiw 
so there are two reason
1. we want to minimize the usage of library/lodash
2. if we use only `lodash.shuffle`, `import shuffle` doesn't work because of the type definition, so this was the only way ='(